www-misc/buku/buku-4.4.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
What's in the way of Python 3.9?

Reply:
upstream doesn't support python3.9 for now

Reply:
But does it work? A lot of upstreams won't specifically mention it for now, and users enabling it know the risks.

Reply:
I was told that the gentoo policy is to follow what upstreams supports (maybe some dev can correct me if I'm wrong). Also python3.9 is in beta.
I suppose we can add python3.9 but for now I don't think we have any reason to do it.

Reply:
There is a broken test with Python 3.9, should probably stay disabled for now, though I've [reported it upstream](https://github.com/jarun/buku/issues/464).